artifactory/utils/buildinfoproperties_test.go

```diff
@@ -21,61 +20,56 @@ const (
-		"artifactory.publish.contextUrl": "http://some.other.url.com",
-		"artifactory.proxy.host":         host,
-		"artifactory.proxy.port":         port,
+		"resolve.contextUrl": "http://some.url.com",
+		"publish.contextUrl": "http://some.other.url.com",
+		"proxy.host":         host,
+		"proxy.port":         port,
 	}
-	createSimplePropertiesFile(t, propertiesFileConfig)
+	createSimplePropertiesFile(t, "map[artifactory.buildInfo.agent.name:/ artifactory.buildInfoConfig.envVarsExcludePatterns:*password*,*psw*,*secret*,*key*,*token* artifactory.buildInfoConfig.includeEnvVars:false artifactory.org.jfrog.build.extractor.maven.recorder.activate:true artifactory.proxy.host:localhost artifactory.proxy.port:8888 artifactory.publish.artifacts:true artifactory.publish.buildInfo:false artifactory.publish.contextUrl:http://some.other.url.com artifactory.publish.filterExcludedArtifactsFromBuild:true artifactory.publish.forkCount:3 artifactory.publish.unstable:false artifactory.resolve.contextUrl:http://some.url.com buildInfo.agent.name:/ buildInfoConfig.envVarsExcludePatterns:*password*,*psw*,*secret*,*key*,*token* buildInfoConfig.includeEnvVars:false org.jfrog.build.extractor.maven.recorder.activate:true proxy.host:localhost proxy.port:8888 publish.artifacts:true publish.buildInfo:false publish.contextUrl:http://some.other.url.com publish.filterExcludedArtifactsFromBuild:true publish.forkCount:3 publish.unstable:false resolve.contextUrl:http://some.url.com]", propertiesFileConfig)
 	setProxy(proxyOrg, t)
 }
 
 func TestCreateSimplePropertiesFileWithoutProxy(t *testing.T) {
 	proxyOrg := getOriginalProxyValue()
 	setProxy("", t)
 	var propertiesFileConfig = map[string]string{
-		"artifactory.resolve.contextUrl": "http://some.url.com",
-		"artifactory.publish.contextUrl": "http://some.other.url.com",
+		"resolve.contextUrl": "http://some.url.com",
+		"publish.contextUrl": "http://some.other.url.com",
 	}
-	createSimplePropertiesFile(t, propertiesFileConfig)
+	createSimplePropertiesFile(t, "map[artifactory.buildInfo.agent.name:/ artifactory.buildInfoConfig.envVarsExcludePatterns:*password*,*psw*,*secret*,*key*,*token* artifactory.buildInfoConfig.includeEnvVars:false artifactory.org.jfrog.build.extractor.maven.recorder.activate:true artifactory.publish.artifacts:true artifactory.publish.buildInfo:false artifactory.publish.contextUrl:http://some.other.url.com artifactory.publish.filterExcludedArtifactsFromBuild:true artifactory.publish.forkCount:3 artifactory.publish.unstable:false artifactory.resolve.contextUrl:http://some.url.com buildInfo.agent.name:/ buildInfoConfig.envVarsExcludePatterns:*password*,*psw*,*secret*,*key*,*token* buildInfoConfig.includeEnvVars:false org.jfrog.build.extractor.maven.recorder.activate:true publish.artifacts:true publish.buildInfo:false publish.contextUrl:http://some.other.url.com publish.filterExcludedArtifactsFromBuild:true publish.forkCount:3 publish.unstable:false resolve.contextUrl:http://some.url.com]", propertiesFileConfig)
```

Comment:
Using one long string of all the props isn't in my opinion the best way to go. It is hard to read and debug.
The props is a map, and I therefore suggest keeping it a map, and not turning the map into one long string.
I'd be happy to review this code again once we find a better way to do this.